src/client/index.tsx

```diff
@@ -127,14 +65,15 @@ const tableElement: JSX.Element = (
             );
         })}
     </div>
-);
+); */
 
-const twoTablesAndHeaderElement = (
+/* const twoTablesAndHeaderElement = (
     <div>
         <h1>Hello</h1>
         {consultantElement}
         {tableElement}
     </div>
 );
+ */
 
-ReactDOM.render(twoTablesAndHeaderElement, document.getElementById("container"));
+ReactDOM.render(<App />, document.getElementById("content"));
```

Comment:
Sorry it's my mistake I forgot that we used "container", please keep using "container" instead of "content".

Reply:
I'm relieved to be not the only one of us who makes mistakes :)